snapshot_types.go

```diff
@@ -2,64 +2,70 @@ package silkworm_go
 
 import "unsafe"
 
-type MemoryMappedRegion struct {
-	FilePath   string
+var NewFilePath = NewCString
```

Comment:
Is it used anywhere?

Reply:
the erigon side uses it to create paths:
https://github.com/erigontech/erigon/pull/13766/files#diff-1cd840b02eba9b3518fc40765ce576fb92ee82dac229573c959010ac1b521d2eR54